AprilSylph/XKit-Rewritten
# New tweak: Hidden following/mutuals indicators

### Description


~~This adds tweaks that remove the bright colors from, and outright hide, the following/mutuals indicators on activity notifications.~~

This adds a tweak that hides the following/mutuals indicators on activity notifications.

Resolves #1122.

### Testing steps


Confirm that the new tweaks remove the bright coloration from or hide the following/mutuals indicators on activity notifications in the activity popover modal.
Confirm that the new tweaks remove the bright coloration from or hide the following/mutuals indicators on activity notifications on the activity page.
